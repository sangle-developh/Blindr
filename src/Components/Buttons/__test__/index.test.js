import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Button from '../index';

it('render button with test id of button', () => {
  render(<Button />);
  const signUpElement = screen.getByTestId('button');
  expect(signUpElement).toBeInTheDocument();
});

it('renders correctly', () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});
