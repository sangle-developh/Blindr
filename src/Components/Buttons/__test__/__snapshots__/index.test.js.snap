// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  data-testid="button"
>
  Hello
</button>
`;
